src/plugins/intel_gpu/src/runtime/ocl/ocl_engine.cpp

```diff
@@ -174,23 +174,41 @@ memory::ptr ocl_engine::allocate_memory(const layout& layout, allocation_type ty
 
     check_allocatable(layout, type);
 
+    auto zero_bytes_layout = false;
+    auto non_zero_layout = layout;
+    if (layout.bytes_count() == 0) {
+        cldnn::layout zero_dim_layout = layout;
+        auto mem_ps = zero_dim_layout.get_partial_shape();
```

Comment:
No need to query real partial shape, just use `{1}`

Reply:
updated